src/View/ViewRenderer.php

```diff
@@ -21,9 +22,9 @@ class ViewRenderer
         'js', 'json', 'xml', 'rss', 'atom', 'txt', 'text', 'html',
     ];
 
-    public function __construct(Factory $viewFactory, BladeCompiler $bladeCompiler, $config = [])
+    public function __construct(Factory $viewFactory, BladeCompiler $bladeCompiler, Collection $config = null)
     {
-        $this->config = collect($config);
+        $this->config = $config ?? collect();
```

Comment:
I wasn't sure what the most idiomatic way to convert this from an array => collection was with the default value. As far as I can tell the only place its ever called without passing a config is in the `ViewRendererTest.php` with `it_does_not_register_view_hint_paths_if_not_specified_in_config` I just didn't want to remove it incase I missed something.